app/sync.js

```diff
@@ -300,9 +300,8 @@ module.exports.init = function (initialState) {
       process.emit(RELOAD_MESSAGE)
     }
   })
-  const isFirstRun = initialized ? false : !initialState.seed && !initialState.deviceId
   ipcMain.on(messages.SYNC_READY, module.exports.onSyncReady.bind(null,
-    isFirstRun))
+    initialized ? false : !initialState.seed && !initialState.deviceId))
```

Comment:
actually this still seems incorrect (assuming `bind` argument expressions are not re-evaluated)